lang/src/test/kotlin/org/partiql/lang/eval/evaluatortestframework/ExprValueStrictEqualsTest.kt

```diff
@@ -9,145 +9,116 @@ import org.partiql.lang.eval.EvaluationSession
 import org.partiql.lang.util.ArgumentsProviderBase
 
 class ExprValueStrictEqualsTest {
-    data class TestCase(val q1: String, val q2: String)
+    data class TestCase(val q1: String, val q2: String, val equals: Boolean)
 
     val ion = IonSystemBuilder.standard().build()
     val pipeline = CompilerPipeline.builder(ion).build()
     val session = EvaluationSession.standard()
 
     @ParameterizedTest
-    @ArgumentsSource(EqualCases::class)
-    fun equalityTests(testCase: TestCase) {
+    @ArgumentsSource(StrictEqualCases::class)
+    fun strictEqualsTests(testCase: TestCase) {
         val v1 = pipeline.compile(testCase.q1).eval(session)
         val v2 = pipeline.compile(testCase.q2).eval(session)
 
-        Assert.assertEquals(true, v1.strictEquals(v2))
+        Assert.assertEquals(testCase.equals, v1.strictEquals(v2))
```

Comment:
My IntelliJ complains about this line, which looks like a complaint about Junit4 and Junit5 being mixed in this file, which does look to me like a legit complaint.